README.md

```diff
@@ -205,6 +205,44 @@ sudo ./scripts/create-installimg.sh \
     8.3:testing install-8.3.testing.img
 ```
 
+### 8.3 updates and linstor
+
+The standalone linstor ISO requires extra packages from a separate
+repository.
+
+```
+./scripts/mirror-repos.sh 8.3 ~/mirrors/xcpng
+
+sudo ./scripts/create-installimg.sh \
+    --srcurl file://$HOME/mirrors/xcpng/8.3 \
+    --output install-8.3.img \
+    8.3:updates
+
+./scripts/create-iso.sh \
+    --srcurl file://$HOME/mirrors/xcpng/8.3 \
+    --output xcp-ng-8.3.linstor.iso \
+    --extra-packages "xcp-ng-linstor" \
+    -V "XCP-NG_830_TEST" \
+    8.3:testing:linstor install-8.3.img
```

Comment:
Why `testing` here as the previous command uses `updates`?

Reply:
fixed